src/utils/junit/JUnitCommandHandler.ts

```diff
@@ -56,6 +56,13 @@ export class JUnitCommandHandler {
 				`${header}${chalk.blue(`${folderMessage} "${item.name}"`)} does not match any test cases`
 			)
 		})
+
+		if (missing.length) {
+			console.error(chalk.yellow('\nTo fix this issue, please rename your test cases in the JUnit file to match the expected format:'))
+			console.error(`  Expected format: ${chalk.green(`${this.project}-XXX: Your test name`)}`)
+			console.error(`  Where XXX is the three-digit test case sequence number.\n`)
```

Comment:
It can be a larger number than 3 digits as well.

So let's just give examples.

Reply:
:) from my experience it has to be at least 3 digits